python/lsst/meas/extensions/gaap/_gaussianizePsf.py

```diff
@@ -120,11 +120,17 @@ def run(self, exposure: lsst.afw.image.Exposure, center: lsst.geom.Point2D,  # n
         targetPsfModel = result.targetPsfModel
         fwhmScience = exposure.getPsf().computeShape(center).getDeterminantRadius()*sigma2fwhm
         fwhmModel = targetPsfModel.computeShape().getDeterminantRadius()*sigma2fwhm
+        self.log.debug(f"Ratio of model to science PSF = {fwhmModel/fwhmScience}")
 
         basisList = makeKernelBasisList(self.kConfig, fwhmScience, fwhmModel,
                                         basisSigmaGauss=basisSigmaGauss,
                                         metadata=self.metadata)
-        spatialSolution, psfMatchingKernel, backgroundModel = self._solve(kernelCellSet, basisList)
+        try:
+            spatialSolution, psfMatchingKernel, backgroundModel = self._solve(kernelCellSet, basisList)
+        except Exception as e:
+            self.log.error("Failed to solve for PSF matching kernel in GAaP for (%f, %f): %s",
+                           (center.getX(), center.getY(), e))
+            raise e
```

Comment:
You need to add this new raise to the docs.  Also, the previous behavior did not include a `raise` statement (even though the docs in `_solve()` say that it does)...please give a thought to what the intended behavior is (i.e. should this really be a `raise` or just a logging of the error and move on?)

Reply:
The modified logging helped uncover a bug in `meas_base` about how undeblended plugins were initialiized. DM-31442 has to be merged before merging this branch to master.

Reply:
Oh, right...somehow I missed the `raise e` in the `_solve()` try block...sorry for the noise!  Can you kick off a fresh Jenkins with the new implementation (which I'll have a look at now).